libbeat/common/kafka/sasl.go

```diff
@@ -0,0 +1,46 @@
+package kafka
```

Comment:
This new file needs the license headers, please run `mage addLicenseHeaders` and commit the changes.